website/api/generating-proofs/local-proving.md

```diff
@@ -6,14 +6,79 @@ The [feature flags] include options to toggle between CPU and GPU proving.
 
 ## Local vs. Remote Proving
 
-For most use cases, we recommend remote proving using [Bonsai].
-The primary reason to consider using local proof generation for your application is to manage private data.
+For most use cases, we recommend remote proving using [Bonsai], our scalable and highly parralel proving service.
 
-> Whoever is generating the proofs can see all private information involved.
+Proving with **private data** is a key reason to consider using local proof generation, as whoever is generating the proofs can see all private information involved.
+With local proof generation your private data never leaves your machine.
 
-_With local proof generation, you can generate proofs about your private data, while your private data never leaves your machine._
+## Proving hardware
+
+RISC Zero's proving system for multiple hardware targets.
+
+:::tip
+In cases where memory is constrained (i.e. less than 10 GB is availble), it may be necessary to change the [segment size limit][segment-limit-docs].
+You can find information about expected memory consumption on our [benchmarks page][datasheet].
+:::
+
+:::info
+The Groth16 prover currently _only_ works on x86 architecture, and so Apple Silicon is _currently unsupported_ (even via Docker).
+
+You can find out more info in the relevant issues [here](https://github.com/risc0/risc0/issues/1520) and [here](https://github.com/risc0/risc0/issues/1749).
+:::
+
+:::note
+When run for the first time, the GPU (e.g. Metal or CUDA) kernels may need to be JIT compiled.
+This can take a few minutes, but should only happen once.
+:::
```

Comment:
one nit: the back to back admonitions are a bit Loud/distracting IMHO, perhaps just one would suffice for the _most_ important thing(s) to highlight?

![image](https://github.com/user-attachments/assets/afa60d9d-8a43-4b02-ab8e-971f8b904976)

Reply:
Good call. In 7ffd2b8bb8ef5802a1824f544d86f1e0f0cb538c I quieted it down a bit.